### PR TITLE
Adding gyptis

### PR DESCRIPTION
**Pull request by benvial**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by benvial** [comment]
@conda-forge/help-python, ready for review.

**Comment by benvial** [comment]
@conda-forge-admin, please restart ci

**Comment by benvial** [comment]
@conda-forge-admin, please ping team

Ready to review

**Comment by benvial** [comment]
@conda-forge-admin, please ping team

File: recipes/gyptis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{% set name = "gyptis" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "f0a2ac4de0e18ee73de6930380caff3e35651faa1cc79baabebddedc910ea33d" %}
```
Comment (ocefpaf) [inline recipes/gyptis/meta.yaml:3]: You don't need a jinja variable for something that us used only once.

File: recipes/gyptis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    - {{ cdt('libxau') }}            # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+  host:
+    - python
+    - pip
+    - fenics
+    - numpy
```
Comment (ocefpaf) [inline recipes/gyptis/meta.yaml:32]: Does this package builds against numpy? If so take a look at: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Comment (benvial) [reply recipes/gyptis/meta.yaml:32]: No it doesn't but fenics does

File: recipes/gyptis/meta.yaml (hunk 1 above)
Comment (benvial) [reply recipes/gyptis/meta.yaml:3]: Okay I will remove the variable

**Pull request merged**